framework/src/media/utils/MediaUtils.cpp

```diff
@@ -307,14 +307,22 @@ bool header_parsing(FILE *fp, audio_type_t audioType, unsigned int *channel, uns
 	unsigned char *header;
 	unsigned char tag[2];
 	bool isHeader;
+	int ret;
 
 	switch (audioType) {
 	case AUDIO_TYPE_MP3:
 		isHeader = false;
-		while (fread(tag, sizeof(unsigned char), 1, fp) > 0) {
-			if (tag[0] == 0xFF) {
+		while (fread(tag, sizeof(unsigned char), 2, fp) > 0) {
+			if ((tag[0] == 0xFF) && ((tag[1] & 0xE0) == 0xE0)) {
```

Comment:
Please add simple description of below logic & mean of 0xE0(line 430 too). Then I will apply it.

Reply:
i added description.